packages/snaps-controllers/src/utils.ts

```diff
@@ -198,3 +200,17 @@ export type Mutable<
 } & {
   [Key in keyof Omit<T, TargetKey>]: T[Key];
 };
+
+/**
+ * Ensures that a relative path starts with `./` prefix.
+ *
+ * @param path - Path to make relative.
+ * @returns The same path, with optional `./` prefix.
+ */
+export function ensureRelative(path: string): string {
+  assert(!path.startsWith('/'));
```

Comment:
This is for normalising `foo/bar.txt` and `./foo/bar.txt`, it existed in a similar way in previous NPM fetching.

It's just a safeguard, it should probably be caught earlier, but if it didn't, it doesn't hurt to have more assert.

Reply:
I don't see how we can ever hit both cases in the if-statement if we have the assertion? One of the cases will fail the assert?

Reply:
[I've added tests for this function](https://github.com/MetaMask/snaps-monorepo/pull/1019/files#diff-8bf9f840796112795f154052e88d801ab1a71e44277e055ffb3ff068a9f223f3R35). Maybe this will clear it's functioning a bit.

Reply:
Yep, thanks! I must have misread 👍